easybuild/easyconfigs/b/BayeScan/BayeScan-2.1-foss-2016a.eb

```diff
@@ -0,0 +1,35 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Pablo Escobar Lopez
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics 
+
+easyblock = 'MakeCp'
+
+name = 'BayeScan'
+version = '2.1'
+
+homepage = 'http://cmpg.unibe.ch/software/BayeScan/'
+description = """BayeScan aims at identifying candidate loci under natural selection from genetic data, 
+ using differences in allele frequencies between populations."""
+
+# BayeScan makefile is hardcoded to use g++. In case you want to use a non-gcc toolchain you will need to patch the makefile 
+toolchain = {'name': 'foss', 'version': '2016a'}
+toolchainopts = {'openmp': True}
+
+source_urls = ['http://cmpg.unibe.ch/software/BayeScan/files/']
+sources = ['%(name)s%(version)s.zip']
+
+prebuildopts = "sed -i.bk 's/g++/${CXX} ${CXXFLAGS}/g' Makefile && "
+
+start_dir = 'source'
+
+files_to_copy = [(['source/bayescan_%(version)s'], 'bin'), 'BayeScan%(version)s_manual.pdf' , 'input_examples', 'R functions']
```

Comment:
please reformat to one file per line:

```python
files_to_copy = [
    (['source/bayescan_%(version)s'], 'bin'),
    'BayeScan%(version)s_manual.pdf',
    'input_examples',
    'R functions',
]
```

also, is that last entry correct? file with a space in its name? :(